flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksIteratorWrapper.java

```diff
@@ -57,7 +57,7 @@ public void seekToFirst() {
 
 	@Override
 	public void seekToLast() {
-		iterator.seekToFirst();
+		iterator.seekToLast();
```

Comment:
@carp84 What's your take on this? 
If this is a valid issue, I guess we should file a proper JIRA and add a test case for this issue?

Reply:
@Myasuka can you take a look?

Reply:
I think you're right to fix this. The reason why this wrong usage did not affect the correctness of state access is that we did not ever use this `seekToLast` in Flink.